CMakeLists.txt

```diff
@@ -73,6 +73,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Options")
 
 set(TILEDB_CMAKE_INPUTS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/cmake/inputs")
+# TBD: consider changing init'ing of TILEDB_CORE_INCLUDE_DIR in various locations
+# to use TILEDB_SOURCE_ROOT_DIR.
+set(TILEDB_SOURCE_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Months ago I put in a definition for `TILEDB_SOURCE_ROOT` in `common.cmake`. It's already got a few uses in various unit/object-libraries. It would be good to unify them rather than to have two closely-named variables that seem to do the same thing.

Reply:
changes using your definition, reserving option of later discussion about whether your approach is best, but less churn for this PR at the moment to use that definition.